src/work-allocation/components/constants/config.constants.ts

```diff
@@ -1,5 +1,6 @@
 import { TaskFieldType, TaskView } from '../../enums';
 import { TaskFieldConfig } from '../../models/tasks';
+import { DERIVED_ICON_CONSTANTS as DIC } from './derived-icon.constants';
```

Comment:
Ha! Might want to use a different exported acronym for this! ;)

Reply:
Good point